src/build/service/docs.js

```diff
@@ -103,14 +103,21 @@ exports.docsTasks = ({tasks, baseDir, spec, cfg, name, cmdOptions, repository, w
         image: nodeImage,
         workingDir: '/app',
         command: ['/app/node_modules/.bin/gulp', 'build-static'],
-        logfile: `${workDir}/yarn-build.log`,
+        logfile: `${workDir}/gulp.log`,
         utils,
         binds: [
           `${appDir}:/app`,
         ],
         baseDir,
       });
 
+      // Streams are wonderful and lead to lots of unhandled promise rejections, meaning that we
+      // do not always get a nonzero exit status when the docs build fails.  This is a good
+      // double-check:
+      if (!fs.existsSync(path.join(staticDir, 'nginx-site.conf'))) {
+        throw new Error('`gulp build-static` did not produce static/nginx-site.conf; check the logfile');
```

Comment:
I've been wondering if we should come up with a Error class that takes "logfile" as an argument and then always tells people where it is. I think we end up with a lot of "check the logfile" errors that don't say where it is.

Just a thought for follow-on work.

Reply:
Yeah, that's a good idea..